osi_sensorview.proto

```diff
@@ -244,60 +249,48 @@ message LidarSensorView
     //
     optional LidarSensorViewConfiguration view_configuration = 1;
 
-    // Ray tracing data.
+    // Received signal element.
     //
-    // This field includes one entry for each ray, in left-to-right,
-    // top-to-bottom order (think of scan lines in a TV).
+    // This field includes one element of the returned signal in the scene, which is backscattered to the sensor.
     //
-    repeated Reflection reflection = 2;
+    // \note Due to e.g. super-sampling or diffuse reflections, a single emitted lidar beam can yield multiple signal returns.
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated ReceivedSignalElement received_signal_element = 2;
 
     //
-    // \brief Definition of the lidar reflection.
+    // \brief Definition of the result of a lidar rendering algorithm.
     //
-    message Reflection
+    message ReceivedSignalElement
     {
-        // Relative signal level of the reflection.
-        //
-        // This takes the signal losses due to scattering and absorption
-        // into account, and will, when multiplied by TX power yield the
-        // potential RX power (disregarding any other RX/TX losses).
-        //
-        // Unit: dB
+        // This message containing the spatial_signal_gain and the correspondig phase.
         //
-        optional double signal_strength = 1;
+        repeated GainPhase gain_phase = 1;
 
         // Time of flight.
         //
-        // This is the time of flight of the reflection, which is directly
-        // proportional to the distance traveled.
+        // This is the time needed by the signal from emission to reception.
         //
         // Unit: s
         //
         optional double time_of_flight = 2;
 
-        // Doppler shift.
+        // Doppler shift
         //
-        // Shift in frequency based on the specified TX frequency.
+        // Shift in frequency based on the emitted frequency.
         //
         // Unit: Hz
         //
         optional double doppler_shift = 3;
 
-        // normal to surface angle.
-        //
-        // The normal of the transmitted beam to the object, roadmarking etc
-        // encounter. \note data is in Lidar coordinate system
-        //
-        // Unit: unit vector
+        // Zero-based index of the emitted lidar beam as configured in
+        // LidarSensorViewConfiguration.
         //
-        optional Vector3d normal_to_surface = 5;
-
-        // ID of the detected object this reflection is associated to.
-        // can be used for raytracing debug
+        // Unit: -
         //
-        // \note ID = MAX(uint64) indicates no reference to an object.
-        optional Identifier object_id = 6;
+        optional uint32 emitted_beam_id = 4;
```

Comment:
This should not be added, because the beam_id only exist in lidar sensors. In the most lidar sensors this is a real output of the sensor. Radar does not have something like real beams because the electromagnetic wave is continious.